packages/sak/sak-historikk/src/HistorikkSakIndex.stories.tsx

```diff
@@ -229,6 +242,13 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
-export const BehandlingIkkeErValgt: Story = {};
+export const BehandlingIkkeErValgt: Story = {
```

Comment:
Du kan heller legga denne under args i meta. Da slepp du å legga den til for kvar story